integration_tests/test_aggregator_batch_commands.py

```diff
@@ -70,12 +71,10 @@ def on_market_cycle(self, market_info):
                             self.errors += 1
             logging.info(f"Batch command placed on the new market")
 
-        for target_market in ["Grid", "House 1", "House 2"]:
-            self.grid_fees_market_cycle_next_market[target_market] = self.calculate_grid_fee("load", target_market)
+        self.require_grid_fees(self.grid_fees_market_cycle_next_market, "current_market_fee")
```

Comment:
Same here to homogenise market naming convention `grid_fees_market_cycle_next_market ` vs  `current_market_fee `

Reply:
https://github.com/gridsingularity/d3a/pull/1083/files/7e06bd6375fc898eea4e0840869abaaa9043680e#r591295763